apps/sertra/components/mod-selector/search.tsx

```diff
@@ -18,6 +18,70 @@ export const SearchBox = <T,>({
 }: SearchBoxProps<T>) => {
   const [isFocused, setFocused] = useState<boolean>(false);
 
+  function focusFirstOption(target: EventTarget & HTMLInputElement) {
+    if (
```

Comment:
Optional chaining operator can be used to call function too, i.e.
`target.parentElement?.parentElement?.nextElementSibling?.children[0]?.children[0]?.focus();`
would remove the need for the `if` block.

That being said, this approach seems really brittle. Small change in the markup will break the function. [useRef](https://reactjs.org/docs/hooks-reference.html#useref) should probably be used here to tag the first option and the search field.

Reply:
Now using refs instead.